check-disk/lib/check-disk.go

```diff
@@ -46,17 +46,13 @@ func checkStatus(current checkers.Status, threshold string, units float64, disk
 		}
 
 		freePct := float64(100) - disk.UsedPercent
-
 		inodesFreePct := float64(100) - disk.InodesUsedPercent
-		if float64(disk.InodesTotal) == float64(0) {
-			inodesFreePct = float64(0)
-		}
 
-		if chkInode && v > inodesFreePct {
+		if chkInode && (disk.InodesTotal != 0 && v > inodesFreePct) {
 			current = status
 		}
 
-		if !chkInode && (v > freePct || v > inodesFreePct) {
+		if !chkInode && (v > freePct || (disk.InodesTotal != 0 && v > inodesFreePct)) {
```

Comment:
I don't think comparing `inodesFreePct` here is correct behavior, since the help says `Exit with WARNING status if less than N units or N% of disk are free`, not N% of inodes.

Reply:
I will propose to change this in another PR. Thanks!